Final text review on pt-br and eng version of C4 Model (#23)
ZupIT/beagle-c4model: en/src/beagle-c4-model.md

# **Overview**

> The software architecture defines the parts of the software and it is also the technology strategy of a project/product. It is very important for the users to see constant updates because, without them, failures may happen.

Author: ```suggestion
The software architecture defines the parts of the software and it is also the technology strategy of a project/product. It is very important for the users to see constant updates because, without them, failures may happen.
```
```suggestion
By definition, software architecture is the technological strategy of a product and / or service, defining the parts that make up a system. And, like every strategy, it is critical that it is constantly updated, after all, a non-viewing planning will tend to be flawed.
```

### **Software documentation**

Documenting a project's architecture takes time, knowledge of tools, and layout techniques. The biggest challenge is to avoid two scenarios:

> * **Scenario 1:** Complex documentation can be confusing and obsolete and it loses the main purpose and it is a waste of time to produce something it won't be used.

Reviewer: ```suggestion
* **Scenario 1:** Complex documentation can be confusing and quickly become obsolete losing its main purpose which is avoiding the waste of time to produce something that won't be used.
```

* **Scenario 2:** Documents with flaws and not enough information.

In both cases, the final result can confuse the user.

The project's architecture is crucial for the user to have a better experience, so it is important to document because it answers questions like:

* How does my system integrate between itself and other systems?
* How can I escalate my application?
* How can I assure my application's security?

> After all this, Beagle's team created a view of our architecture in a simpler way, the C4 Model.

Reviewer: I wonder if instead of 'our' the word 'its' would be better. What do you think?

Author: ```suggestion
After all this, Beagle's team created a view of its architecture in a simpler way, the C4 Model.
```

Author: I agree, @hectorcustodiozup ! Just changed it here.
